api/app/Models/Experience.php

```diff
@@ -35,13 +35,14 @@ class Experience extends Model
     /**
      * Create a new concrete model instance that is existing, based on the type field.
      *
-     * @param  object  $attributes
+     * @param  mixed  $attributes
      * @param  string|null  $connection
      * @return static
      */
     public function newFromBuilder($attributes = [], $connection = null)
     {
-        $model = $this->newInstanceFromType($attributes->experience_type);
+        /** @disregard Even though it is typed as an array, it is actually a class */
+        $model = $this->newInstanceFromType(((object) $attributes)->experience_type);
```

Comment:
This is the confusing bit. The `Model` class this extends expected `array $attributes = []` but we actually get an `object` so the cast works but is a bit hacky :cry: